Library/Homebrew/test/support/helper/spec/shared_context/homebrew_cask.rb

```diff
@@ -31,11 +32,18 @@
         FileUtils.ln_sf TEST_FIXTURE_DIR.join("cask"), tap.path
       end
 
+      external_caskroom_tap.tap do |tap|
+        FileUtils.mkdir_p tap.path.dirname
+        FileUtils.ln_sf TEST_FIXTURE_DIR.join("external-cask"), tap.path
+      end
+
       example.run
     ensure
       FileUtils.rm_rf dirs
       Hbc.default_tap.path.unlink
       FileUtils.rm_rf Hbc.default_tap.path.parent
+      external_caskroom_tap.path.unlink
```

Comment:
This is unnecessary when you are already calling `FileUtils.rm_rf` on the parent below.